File: csharp/sources/Valkey.Glide.InterOp/Errors/TimeoutException.cs
namespace Valkey.Glide.InterOp.Errors;

/// <summary>
/// A timeout from Glide to Valkey service that is thrown when a request times out.
/// </summary>
public sealed class TimeoutException : GlideException
{
    public TimeoutException() { }

    public TimeoutException(string message) : base(message) { }

    public TimeoutException(string message, Exception innerException) : base(message, innerException) { }
}

File: java-old/client/src/main/java/glide/api/models/exceptions/TimeoutException.java
/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
package glide.api.models.exceptions;

/** Timeout error: Errors that are thrown when a request times out. */
public class TimeoutException extends GlideException {
    public TimeoutException(String message) {
        super(message);
    }
}
